dispatcher/DAQController.py

```diff
@@ -67,13 +68,10 @@ def SolveProblem(self, latest_status, goal_state):
                 run in the neutron veto but it is already running a combined run and
                 therefore unavailable. The frontend should prevent many of these cases though.
 
-        The way that works is this. We do everything iteratively. Like if we see that
-        some detector needs to be stopped in order to proceed we issue the stop command
-        then move on. Everything is then re-evaluated once that command runs through.
-
-        I wrote this very verbosely since it's got quite a few different possibilities and
-        after rewriting once I am convinced longer, clearer code is better than terse, efficient
-        code for this particular function. Also I'm hardcoding the detector names. 
+        The way that works is this:
```

Comment:
minor: There is definitely something to be said for a max-line length. Perhaps make sure you have no more than ~100 characters of however many is convention for redax for readability.